extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisClientName.java

```diff
@@ -26,7 +26,7 @@
  *     RedisClient client;
  * </pre>
  */
-@Target({ ElementType.TYPE, ElementType.METHOD, ElementType.FIELD })
+@Target({ ElementType.TYPE, ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER })
 @Retention(RUNTIME)
 @Documented
 @Qualifier
```

Comment:
We need @mkouba opinion. The last time I used a qualifier, I heard him shouting at me and I'm living more than 1000 km away from him :-)

It seems ok here, but better check.

Reply:
Hmm, this is OK. This is what we use everywhere for named whatever.

Reply:
This should be fine as the annotation is used as a CDI qualifier (@cescoffier do you spot the difference? Also I don't remember this incident and BTW you live approx. 1000km away from me, or? ;-)